.github/workflows/webviz-subsurface-components.yml

```diff
@@ -102,24 +102,50 @@ jobs:
           npm run test_correctness --prefix ./react
           pytest ./tests --headless
 
-      # Run cypress tests on firefox browser
-      # - name: E2E on firefox
-      #   if: github.event_name != 'release' && matrix.e2e-on-firefox
-      #   uses: cypress-io/github-action@33ec2f03412577367c7c6aac7b646c88e646e33c
-      #   with:
-      #     browser: firefox
-      #     working-directory: ./react
-      #     start: npm run storybook
-      #     wait-on: "http://localhost:6006"
-      # - uses: actions/upload-artifact@v2
-      #   if: failure()
-      #   with:
-      #     name: cypress-screenshots
-      #     path: |
-      #       react/cypress/screenshots
-      #       react/cypress/snapshots
-      #       !react/cypress/snapshots/*/*.snap.png
-      #     retention-days: 5
+      # This action allows caching cypress dependencies
+      - name: Cache Cypress
+        if: github.event_name != 'release' && matrix.e2e-tests
+        id: cache-cypress
+        uses: actions/cache@v1
+        with:
+          # list of files to cache and restore
+          path: ~/.cache/Cypress
+          # An explicit key for restoring and saving the cache
+          key: cypress-cache-v2-${{ runner.os }}-${{ hashFiles('**/package.json') }}
+
+      # now let's install Cypress binary
+      - name: install cypress binary
+        if: github.event_name != 'release' && matrix.e2e-tests
+        run: |
+          cd react
+          npx cypress install
+
+      # run the cypress tests
+      - name: e2e on firefox
+        if: github.event_name != 'release' && matrix.e2e-tests
+        run: |
+          mkdir clone
+          cd clone
+          git clone --branch cypress-snaps --depth 1 --filter=blob:none https://github.com/avanti-em/webviz-subsurface-components
```

Comment:
Should this be changed to official repo before merge?

Reply:
Oops! I forgot to change it. Thanks for catching.